listeners/websocket.js

```diff
@@ -0,0 +1,64 @@
+var WebSocketClient = require('websocket').client;
```

Comment:
Why var twice. Once (with commas) or three times (with semicolons in each line) would be normal.